wdl-grammar/tests/parsing/reserved-meta-names/source.tree

```diff
@@ -0,0 +1,49 @@
+RootNode@0..175
+  Comment@0..56 "# This is a test of r ..."
+  Whitespace@56..58 "\n\n"
+  VersionStatementNode@58..69
+    VersionKeyword@58..65 "version"
+    Whitespace@65..66 " "
+    Version@66..69 "1.1"
+  Whitespace@69..71 "\n\n"
+  TaskDefinitionNode@71..174
+    TaskKeyword@71..75 "task"
+    Whitespace@75..76 " "
+    Ident@76..80 "test"
+    Whitespace@80..81 " "
+    OpenBrace@81..82 "{"
+    Whitespace@82..87 "\n    "
+    MetadataSectionNode@87..124
+      MetaKeyword@87..91 "meta"
+      Whitespace@91..92 " "
+      OpenBrace@92..93 "{"
+      Whitespace@93..102 "\n        "
+      MetadataObjectItemNode@102..118
+        VersionKeyword@102..109 "version"
```

Comment:
I imagine I'm misunderstanding something about `rowan` or `logo`, but shouldn't the string `version` be parseable as an `Ident` `Token`? That's what it is in _this_ context, and `version` matches the regex used for `Ident` (`[a-zA-Z][a-zA-Z0-9_]*`).

Reply:
That's a good point; let me figure out a way to substitute an ident token here, which will also simplify the AST as it then doesn't have to search for the reserved keyword tokens either.